README.md

```diff
@@ -61,7 +61,7 @@ See [USECASES.md](USECASES.md)
 The specification allows for a no-code implementation as a collection of files served statically (e.g. in a cloud bucket, or a Git repository). To do this, you need the following JSON files:
 
 - ```datasets```: served in response to ```GET /datasets```
-- ```dataset/{id}```: served in response to ```GET /datasets/{id}```.  e.g. a dataset with id ```mydataset``` should have a corresponding file ```/datasets/mydataset```
+- ```dataset/{id}```: served in response to ```GET /dataset/{id}```.  e.g. a dataset with id ```mydataset``` should have a corresponding file ```/datasets/mydataset```
```

Comment:
The last part of this line should say ```/dataset/mydataset``` instead no?

Reply:
Yes, thank you! I've pushed a fix for the fix.